extensions/fetch/21_formdata.js

```diff
@@ -332,37 +330,34 @@
         ["Content-Type", type || "application/octet-stream"],
       ];
       return this.#writeHeaders(headers);
-    };
-
-    /**
+    } /**
      * @param {string} field
      * @returns {void}
      */
-    #writeFieldHeaders = (field) => {
+
```

Comment:
Are these changes intentional? Looks like comment formatting breaks

Reply:
hmm sort of `deno fmt` strangeness